### ISSUE TITLE
Unexpected behavior with comparisons()

### ISSUE THREAD

**Issue by arthur-albuquerque** [vincentarelbundock/marginaleffects#426]
I noticed that comparisons() was not generating the appropriate output when the exposure variable was coded as numeric. When numeric, the Contrast output regards "(x + 1) - x". When factor, "1 - 0". 

I was expecting this behavior with marginaleffects(), not comparisons(). 

Is this a bug?

I have {marginaleffects} v0.6.0 installed.

``` r
library(marginaleffects)
library(haven)

data = read_dta(url("https://github.com/migariane/TutorialComputationalCausalInferenceEstimators/raw/main/rhc.dta"))
# Define the outcome (Y), exposure (A), confounder (C)
# Set all as numeric

data$Y  <- data$death_d30
data$Y <- as.numeric(data$Y)
data$A  <- data$rhc
data$A <- as.numeric(data$A)
data$C  <- data$sex
data$C <- as.numeric(data$C)

# A as numeric
glm(Y ~ A + C, family="binomial", data=data)  |> 
    comparisons(variables = "A") |> 
    summary()
#> Average contrasts 
#>   Term    Contrast  Effect Std. Error z value   Pr(>|z|)   2.5 %  97.5 %
#> 1    A (x + 1) - x 0.07228    0.01248   5.791 6.9981e-09 0.04782 0.09674
#> 
#> Model type:  glm 
#> Prediction type:  response

# Note `Contrast` column: "(x + 1) - x"

# A as factor
data$A <- as.factor(data$A)

glm(Y ~ A + C, family="binomial", data=data)  |> 
    comparisons(variables = "A") |> 
    summary()
#> Average contrasts 
#>   Term Contrast  Effect Std. Error z value   Pr(>|z|)  2.5 %  97.5 %
#> 1    A    1 - 0 0.07351    0.01296   5.671 1.4237e-08 0.0481 0.09892
#> 
#> Model type:  glm 
#> Prediction type:  response

# Note `Contrast` column: "1 - 0"
```

<sup>Created on 2022-07-26 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by ngreifer** [comment]
This was just fixed in #416. Try installing the newest version and see if the problem persists.

**Comment by arthur-albuquerque** [comment]
Thanks, Noah. Just updated it to v0.6.0.9, and it works as expected.
